pyproject.toml

```diff
@@ -1,70 +1,82 @@
-[tool.poetry]
+[project]
 name = "python-kasa"
 version = "0.7.2"
-description = "Python API for TP-Link Kasa Smarthome devices"
-license = "GPL-3.0-or-later"
-authors = ["python-kasa developers"]
-repository = "https://github.com/python-kasa/python-kasa"
+description = "Python API for TP-Link Kasa and Tapo devices"
+license = {text = "GPL-3.0-or-later"}
+authors = [ { name = "python-kasa developers" }]
 readme = "README.md"
-packages = [
-  { include = "kasa" }
+requires-python = ">=3.9,<4.0"
+dependencies = [
+  "asyncclick>=8.1.7",
+  "pydantic>=1.10.15",
+  "cryptography>=1.9",
+  "async-timeout>=3.0.0",
+  "aiohttp>=3",
+  "typing-extensions>=4.12.2,<5.0",
 ]
-include = [
-  { path= "CHANGELOG.md", format = "sdist" }
+
+classifiers = [
+    "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
```

Comment:
With only month for the 3.13 release, should/can we already include it here?

Reply:
Sure makes sense

Reply:
That's done